src/Symfony/Component/VarExporter/LazyObjectInterface.php

```diff
@@ -16,7 +16,7 @@ interface LazyObjectInterface
     /**
      * Returns whether the object is initialized.
      */
-    public function isLazyObjectInitialized(): bool;
+    public function isLazyObjectInitialized(bool $partial = false): bool;
```

Comment:
The name `$partial` might be misleading. `$a->isLazyObjectInitialized(partial: true)` will still return `true` for a fully initialized object.

Reply:
I figured a proper explanation, see docblock